tests/test_tuple_metaprogramming.cpp

```diff
@@ -1,151 +1,165 @@
+        static_assert(std::is_same_v<type, std::tuple<int, long>>);
     }
     {
-        using type = wmtk::utils::metaprogramming::tuple::remove_void_types_t<void,int,void,long>;
-        static_assert(std::is_same_v<type,std::tuple<int,long>>);
+        using type =
+            wmtk::utils::metaprogramming::tuple::remove_void_types_t<void, int, void, long>;
+        static_assert(std::is_same_v<type, std::tuple<int, long>>);
     }
-
 }
 TEST_CASE("test_get_unique_metaprogramming", "[tuple]")
 {
-
     {
         using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<>;
-        static_assert(std::is_same_v<type,std::tuple<>>);
+        static_assert(std::is_same_v<type, std::tuple<>>);
     }
     {
         using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<int>;
-        static_assert(std::is_same_v<type,std::tuple<int>>);
+        static_assert(std::is_same_v<type, std::tuple<int>>);
     }
     {
-        using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<int,int>;
-        static_assert(std::is_same_v<type,std::tuple<int>>);
+        using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<int, int>;
+        static_assert(std::is_same_v<type, std::tuple<int>>);
     }
     {
-        using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<int,int,int>;
-        static_assert(std::is_same_v<type,std::tuple<int>>);
+        using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<int, int, int>;
+        static_assert(std::is_same_v<type, std::tuple<int>>);
     }
     {
-        using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<int,long,int>;
-        static_assert(std::is_same_v<type,std::tuple<long,int>>);
+        using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<int, long, int>;
+        static_assert(std::is_same_v<type, std::tuple<long, int>>);
     }
     {
-        using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<int,long,long>;
-        static_assert(std::is_same_v<type,std::tuple<int,long>>);
+        using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<int, long, long>;
+        static_assert(std::is_same_v<type, std::tuple<int, long>>);
     }
     {
-        using type = wmtk::utils::metaprogramming::tuple::get_unique_types_t<long,float,int,long,long,long,int>;
-        static_assert(std::is_same_v<type,std::tuple<float,long,int>>);
+        using type = wmtk::utils::metaprogramming::tuple::
+            get_unique_types_t<long, float, int, long, long, long, int>;
+        static_assert(std::is_same_v<type, std::tuple<float, long, int>>);
     }
-
 }
 TEST_CASE("test_get_unique_remove_void_types_metaprogramming", "[tuple]")
 {
-
     {
         using type = wmtk::utils::metaprogramming::tuple::get_unique_remove_void_types_t<>;
-        static_assert(std::is_same_v<type,std::tuple<>>);
+        static_assert(std::is_same_v<type, std::tuple<>>);
```

Comment:
I am not sure if `static_assert` is the best choice in a test. I think it might actually be better to have those as `CHECK`s.

Reply:
static_assert are compile time checks - so there is no need to actually use CHECK / i'm using these unit tests as placeholders for things that will just fail to compile